language/move-lang/functional-tests/tests/0L/autopay/autopay_pledge_types.move

```diff
@@ -0,0 +1,370 @@
+//! account: bob, 10000GAS, 0, validator
```

Comment:
Can we change this filename and the keyword 'pledge' for 'instruction' throughout?

Reply:
Changed pledge -> instruction her and in a couple other places.